easybuild/easyconfigs/t/task-spooler/task-spooler-1.0.2-GCCcore-11.3.0.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'task-spooler'
+version = '1.0.2'
+
+homepage = 'http://viric.name/soft/ts/'
+description = 'task spooler is a Unix batch system where the tasks spooled run one after the other.'
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = [homepage]
+sources = ['ts-%(version)s.tar.gz']
+checksums = ['f73452aed80e2f9a7764883e9353aa7f40e65d3c199ad1f3be60fd58b58eafec']
+
+dependencies = {
+    ('binutils', '2.38'),
+}
```

`binutils` is only needed for building:
```suggestion
builddependencies = [('binutils', '2.38')]
```